src/audio/module_adapter/module/dts.c

```diff
@@ -116,8 +131,6 @@ static int dts_effect_populate_buffer_configuration(struct comp_dev *dev,
 	buffer_config->sampleRate = rate;
 	buffer_config->numChannels = channels;
 	buffer_config->periodInFrames = dev->frames;
-	/* totalBufferLengthInBytes will be populated in dtsSofInterfacePrepare */
-	buffer_config->totalBufferLengthInBytes = 0;
```

Comment:
same here? Is this related to the free memory callback?

Reply:
you can treat it as code update for library uprev, see my comment https://github.com/thesofproject/sof/pull/6587#issuecomment-1314793068

Reply:
> No, this line is not related to the memory callback. We've found it's redundant here and removed it. Should we separate it to another commit or PR?

yes, please

Reply:
@ranj063 I've separated it to another commit. Please let me know if it's good now. Thanks.